### ISSUE TITLE
Code doesn't regenerate - FreeOTP Version 1.99 (19)

### ISSUE THREAD

**Issue by MikeDawg** [freeotp/freeotp-android#195]
Hi Nathan,

I'm working off of FreeOTP Version 1.99 (19). The beta project. Just wanted to report some bugs

In comparison to the "older" version of FreeOTP, it would be able to "re-generate" a new code, most of the times, depending on key time length etc, just by clicking on the key. That is not the case with this version, and there is no way to "ask" for a new key.

**Comment by npmccallum** [comment]
By "regenerate a new code" I presume you mean "get the next code"? If I understand this correctly, you mean you cannot get the next code until the previous code expires. Is that correct?

**Comment by MikeDawg** [comment]
Correct, I know there used to be *some* keys you could regenerate/get next code, if you click on the displayed number, and once it got down to the last ~20 seconds of that key being valid.

**Comment by jimmysjolund** [comment]
This is the behaviour in 2.0.5, closing.
